set.mm

```diff
@@ -530899,6 +530841,21 @@ extends this idea to multiple additions (TODO).
       ( weq wn wex wal ax13lem2 ax13lem1 syld ) ABDECBDZAFKKAGABCHABCIJ $.
   $}
 
+  ${
+    wl-axc11rALT.1 $e |- ( A. y y = x -> ( A. y y = x -> A. x y = x ) ) $.
+    wl-axc11rALT.2 $e |- ( A. x x = y -> ( A. x ph -> A. y ph ) ) $.
+    $( Proving ~ axc11 from ~ axc11r (in the end from ~ ax-12 ) is somewhat
+       tricky and requires ~ ax-10 and ~ ax-13 .  The reverse direction is
+       rather simple as demonstrated here.  We provide the two instances of
+       ~ axc11 used in the proof here as hypotheses.  Note that besides these
+       references to ~ ax-12 no other axiom beyond ~ ax-7 is used.
+       (Contributed by Wolf Lammen, 18-Jul-2023.)  (New usage is discouraged.)
+       (Proof modification is discouraged.) $)
+    wl-axc11rALT $p |- ( A. y y = x -> ( A. x ph -> A. y ph ) ) $=
+      ( weq wal wi pm2.43i equcomi alimi 3syl ) CBFZCGZMBGZBCFZBGABGACGHNODIMPB
+      CBJKEL $.
```

Comment:
I think this could go to Main, somewhere between ax-7 and ax-8, with label ~axc11rc11 (since I think it is the usual labeling convention: when proving axiom A from axiom B, label it axAB). Then, the third sentence of the comment becomes unnecessary, and I would reverse the beginning of the comment: in the first sentence, describe what you prove ("Prove ~axc11 from ~axc11r."), and then, comment that the reverse direction is harder. Probably, then, no need of any discouragement tag.